teamsupport/services.py

```diff
@@ -63,8 +93,9 @@ def update_ticket(self, ticket_id, data):
             data=data, root='Ticket', send_as_xml=True)
         return self.parse_xml_response(response)
 
-    def get_ticket_actions(self, ticket_id):
-        response = self.get('Tickets/{0}/Actions'.format(ticket_id))
+    def get_ticket_actions(self, ticket_id, query_params=None):
```

Comment:
In some places we accept query_params as `**query_params` and in others like `query_params=None`.  Probably it makes sense to be consistent?